Sitemap.md

```diff
@@ -0,0 +1,64 @@
+# Sitemap Generation
```

Comment:
I don't think this makes sense anymore now that plugs and Lua only runs in the browser. It could be rewritten to generate a `sitemap.xml` file in your space, but not be served using an event like at is implemented now.